package/libpiksi/libpiksi/src/util.c

```diff
@@ -833,52 +883,29 @@ pipeline_t *create_pipeline(void)
   return pipeline;
 }
 
-bool str_digits_only(const char *str)
+void print_trace(const char *assert_str, const char *file, const char *func, int lineno)
 {
-  if (str == NULL) {
-    return false;
-  }
-
-  /* Empty string */
-  if (!(*str)) {
-    return false;
-  }
+  void *array[32];
+  size_t size;
+  char **strings;
+  size_t i;
 
-  while (*str) {
-    if (isdigit(*str++) == 0) {
-      return false;
-    }
-  }
+  size = backtrace(array, 32);
+  strings = backtrace_symbols(array, size);
```

Comment:
Check for `NULL`. Function doc states `The return value is NULL if sufficient memory for the strings cannot be obtained`.